### ISSUE TITLE
Anteproyecto versión 2

### ISSUE THREAD

**Issue by vickidemaio** [fiuba-labo-de-micro-miercoles/2019-2c-trabajo-practico-Pochoclera#2]
Buenas,
Subimos una nueva versión del anteproyecto con las correcciones señaladas.

Saludos!

**Comment by fbaglivo** [comment]
Estimados, falta el diagrama de flujos del firmware. 
Tienen dos issues abiertos para seguir la corrección del anteproyecto. Cierren el anterior y sigamos en este

**Comment by ndirenzo** [comment]
Hola Fabricio. Habíamos quedado en la clase de ayer con Arias que el diagrama de flujos lo podíamos agregar mas adelante, ya que todavía no tenemos claro como realizarlo

**Comment by fbaglivo** [comment]
Perfecto.

**Comment by fbaglivo** [comment]
Hola! Seria bueno que ya vayan haciendo un primer draft del diagrama de flujos. También deben agregar la fuente al diagrama de flujo. Por último, sólo dejen la versión mas reciente del anteproyecto.

**Comment by fbaglivo** [comment]
Hola! Seria bueno que ya vayan haciendo un primer draft del diagrama de flujos. También deben agregar la fuente al diagrama de flujo. Por último, sólo dejen la versión mas reciente del anteproyecto.

**Comment by fbaglivo** [comment]
y? Vayan documentado lo que planean hacer (y lo que ya hicieron)

**Comment by ndirenzo** [comment]
Frabricio, estuvimos un poco complicados la semana pasada y no pudimos avanzar con el diagrama de flujos. Prometemos arrancarlo en estos dias!
En cuanto a la documentación donde seria correcto ir actualizando? En la siguiente ruta: **2019-2c-trabajo-practico-Pochoclera/doc/reportes/leeme.txt**?

**Comment by fbaglivo** [comment]
No, les diria que hagan un archivo para el diagrama de bloques, uno para el de flujo del firmware y otro para el circuito electrico y los suban. La idea es que los vayan actualizando a medida que agregan el hardware,
